File: KyrgyzTestFrontend/src/types/certificateRecord.ts
/** Запись реестра сертификатов, возвращаемая CertRecordController. */
export interface CertificateRecord {
    /** Идентификатор записи для просмотра и удаления. */
    id: number;
    /** ФИО получателя сертификата. */
    received: string;
    /** Организация получателя, если она была указана при импорте. */
    organization: string | null;
    /** Уникальный номер сертификата. */
    certificateNumber: string;
    /** Уровень сертификата, если он передан в исходном файле. */
    level: string | null;
    /** Дата выдачи в ISO-формате, которую возвращает API. */
    issueDate: string;
    /** Дополнительные сведения из реестра. */
    additionalInfo: string | null;
}

File: KyrgyzTestBackEnd/KyrgyzTest.Core/Models/CertificateRecords/CertificateRecord.cs
using System.Runtime.InteropServices.JavaScript;
using KyrgyzTest.Core.Abstractions;

namespace KyrgyzTest.Core.Models.CertificateRecords;

public class CertificateRecord : Entity
{
    public required string Received { get; set; }
    
    public string? Organization { get; set; }

    public required string CertificateNumber { get; set; }
    
    public string? Level { get; set; }
    
    public DateTime IssueDate { get; set; }
    
    public string? AdditionalInfo { get; set; }

}